modules/tournament/src/main/TournamentShield.scala

```diff
@@ -1000,8 +1000,7 @@ object TournamentShield {
         extends Category(
           Variant.Abalone(strategygames.abalone.variant.Abalone),
           Blitz53,
-          19,
-          1
+          2
```

Comment:
You need to bring back line 1004, otherwise this will schedule the tournament at the same time as Chess960.
So
```
          Variant.Abalone(strategygames.abalone.variant.Abalone),
          Blitz53,
          2,
          1
```